libs/platform/src/lib/components/form/select/select.component.ts

```diff
@@ -2,25 +2,35 @@ import {
     Component,
     OnInit,
     Input,
-    forwardRef,
     ChangeDetectionStrategy,
     ChangeDetectorRef,
     ContentChild,
     TemplateRef,
     AfterContentInit,
-    ElementRef
+    ElementRef,
+    NgZone,
+    Optional,
+    Attribute,
+    Self,
+    Injector
 } from '@angular/core';
-import { NG_VALUE_ACCESSOR } from '@angular/forms';
-import { DynamicComponentService, SelectComponent as fdSelect, SelectProxy } from '@fundamental-ngx/core';
+import { NgControl } from '@angular/forms';
+import { ViewportRuler } from '@angular/cdk/scrolling';
+import { Directionality } from '@angular/cdk/bidi';
+import { LiveAnnouncer } from '@angular/cdk/a11y';
+
+import { DynamicComponentService, SelectComponent as fdSelect, DialogConfig} from '@fundamental-ngx/core';
+
+import { FormFieldControl } from '../form-control';
 
 @Component({
     selector: 'fdp-select',
     templateUrl: './select.component.html',
     styleUrls: ['./select.component.scss'],
     providers: [
         {
-            provide: NG_VALUE_ACCESSOR,
-            useExisting: forwardRef(() => SelectPlatformComponent),
+            provide: FormFieldControl,
+            useExisting: SelectPlatformComponent,
```

Comment:
We dont prefix component with platform , this is only for Module. it shoudl be just SelectComponent

Reply:
Sure @Frank, (this is taken care) all platform related changes will be pushed in another PR. as name change was leading to push more files from platform. I kept it as it is.